### ISSUE TITLE
[Crash] System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service)

### ISSUE THREAD

**Issue by postpersonality** [sn4k3/UVtools#804]
### System

```Text
UVtools v4.0.5 X64
Operative system: Linux 6.5.5-200.fc38.x86_64 #1 SMP PREEMPT_DYNAMIC Sun Sep 24 15:52:44 UTC 2023 X64
Processor: AMD Ryzen 9 5900HX with Radeon Graphics
Processor cores: 16
Memory RAM: 29.37 / 30.74 GB
Runtime: fedora.38-x64
Framework: .NET 6.0.25
AvaloniaUI: 11.0.6
OpenCV: 4.8.1

Path:       /tmp/.mount_gearlexsaXgy/usr/bin/
Executable: /tmp/.mount_gearlexsaXgy/usr/bin/UVtools
```

### Printer and Slicer

- Printer: 
- Slicer: 

### Description of the bug

```
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service)
 ---> Tmds.DBus.Protocol.DBusException: org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service
   at Tmds.DBus.Protocol.DBusConnection.MyValueTaskSource`1.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at Tmds.DBus.Protocol.DBusConnection.CallMethodAsync(MessageBuffer message)
   at Tmds.DBus.Protocol.Connection.CallMethodAsync(MessageBuffer message)
   --- End of inner exception stack trace ---
Category: Task
File: UVtools_demo_file.sl1  [Version: 0] [Class: SL1File]

Machine date time: 12/15/2023 2:28:30PM
    UTC date time: 12/15/2023 11:28:30AM
```

### How to reproduce

I just opened the app the first time, opened up settings (File/Settings), clicked 2nd tab (Do not remember the name)

### Files

I haven't opened any file

**Comment by sn4k3** [comment]
Duplicate of #803
